lib/paper_trail/reifier.rb

```diff
@@ -60,10 +61,14 @@ def reify(version, options)
       private
 
       # Set all the attributes in this version on the model.
-      def reify_attributes(model, version, attrs)
+      def reify_attributes(model, version, attrs, options)
         enums = model.class.respond_to?(:defined_enums) ? model.class.defined_enums : {}
         model.class.unserialize_attributes_for_paper_trail! attrs
 
+        # Reifies habtm if the option is passed, otherwise sets the habtm
+        # attributes to nil so they are not reified
+        reify_has_and_belongs_to_many(model, attrs, options)
```

Comment:
As the join model can't be versioned, the opposing habtm models ID is stored instead as an attribute in the serialised object. So `has_and_belongs_to_many :bars` would be saved as `bar_ids = [1, 2, 3, etc.]` in the serialised object column of the versions table. This means that upon reification of attributes, `reify_attributes` would attempt to set `model.bar_ids = [1, 2, 3]` even if the option to reify habtm is set to false. So these keys must be deleted from the attrs hash (they are deleted, not set to nil, my mistake)

So if the habtm option is not passed, the model retains the habtm associations from the latest version.